docs/guide/opentelemetry-elastic.asciidoc

```diff
@@ -0,0 +1,181 @@
+before sending the data to the Elastic Stack.
+By extending the OpenTelemetry collector,
+no changes are needed in your instrumented services in order to begin using the Elastic Stack.
+
+*************
+Instrumented with
+OpenTelemetry
+    ↓
+    ↓                       OpenTelemetry Collector                   The Elastic Stack [ESS]
+Service 1  --->  ┐            ┌────────────────┐            ┌─────────────────────────────────────────┐
+Service 2  --->  ┤     --->   |Elastic Exporter|    --->    |APM Server --> Elasticsearch --> APM app)|
+Service 3  --->  ┘            └────────────────┘            └─────────────────────────────────────────┘
+                        ↑                            ↑
+                        ↑                            ↑
+                   OpenTelemetry                  Elastic
+                   Protocol                       Protocol
+*************
+
+[float]
+[[open-telemetry-elastic-works]]
+=== How the OpenTelemetry Collector works
+
+The OpenTelemetry collector uses three different types of components to handle data: `receivers`, `processors`, and `exporters`.
+
+* `receivers`: Configure how data gets to the collector. At least one receiver must be configured.
+* `processors`: Defines optional transformations that occurs between receiving and exporting data.
+* `exporters`: Configures how data is sent to its destination--in this case, the Elastic Stack.
+
+Once a `receiver`, `processor`, and `exporter` is defined, `pipelines` can be configured in the `services` section of your configuration. Specifically, a `traces` pipeline will define the path of trace data through your collector, and bring all three of these components together.
+
+TIP: More information is available in the
+{ot-pipelines}[OpenTelemetry pipeline docs]
+
+A final note: `extensions` can also be enabled for tasks like monitoring your collectors health.
+See the {ot-extension}[OpenTelemetry extension readme]
+for a list of supported extensions.
+
+[[open-telemetry-elastic-get-started]]
+=== Get started
+
+NOTE: This guide assumes you've already instrumented your services with the OpenTelemetry API and/or SDK.
+If you haven't, see the Elastic APM <<install-and-run,install and run guide>> to get started with Elastic APM Agents instead.
+
+==== Deployment planning
+
+OpenTelemetry Collectors can be run as an Agent, or as standalone collectors.
+They can be deployed as often as necessary and scaled up or out.
+
+Deployment planning resources are available in OpenTelemetry's {ot-collector}[Getting Started]
+documentation, and {ot-scaling}[Collector Performance] research.
+
+==== Download the collector
+
+The Elastic exporter lives in the {ot-repo}[`opentelemetry-collector-contrib repository`],
+and the latest release can be downloaded from {ot-repo}/releases[GitHub releases page].
+You can also build the collector-contrib repository by cloning it and running:
+
+[source,bash]
+----
+make otelcontribcol
```

Comment:
I expect the number of people cloning and building the repo would be in the minority, over time anyway. Perhaps emphasise the Docker image by putting it first?